rotkehlchen/exchanges/binance.py

```diff
@@ -338,12 +339,14 @@ def api_query(
 
             api_subdomain = api_type if is_new_futures_api else 'api'
             request_url = (
-                f'https://{api_subdomain}.{self.uri}{api_type}/v{api_version!s}/{method}?'
+                f'https://{api_subdomain}.{self.uri}{api_type}/v{api_version!s}/{method}'
             )
-            request_url += urlencode(call_options)
             log.debug(f'{self.name} API request', request_url=request_url)
             try:
-                response = self.session.get(request_url, timeout=timeout)
+                if request_type == 'get':
+                    response = self.session.get(url=request_url, params=call_options, timeout=timeout)  # noqa: E501
+                else:  # post
+                    response = self.session.post(url=request_url, data=call_options, timeout=timeout)  # noqa: E501
```

Comment:
you don't need this. Rename "request_type" above to `method` as this is the proper variable name and use it.

```suggestion
                response = self.session.request(method=method, url=request_url, params=call_options, data=call_options, timeout=timeout)
```

You mau still need a further tweak to choose either params or data ... if that's needed I would also put that as a function argument.